src/components/Empty/rtl.scss

```diff
@@ -0,0 +1,5 @@
+.empty {
+    &-rtl {
```

Comment:
interesting. looking forward to seeing this in action

Reply:
Ya, we'll need to add the rtl SASS file to each component, start ad-hoc testing to get ahead of future support of AR, HE.